README.md

```diff
@@ -77,17 +77,17 @@ make help
 
 Before committing the code, please format it and check if it passes all tests using
 ```bash
-DEVMODE=LOCAL make format clean mod deps generate qa example
+DEVMODE=LOCAL make x
```

Comment:
This jarred a little, I saw it's in the makefile, but I wondered if it was a typo :)